modules/ctrl_tcp/ctrl_tcp.c

```diff
@@ -200,11 +202,15 @@ static bool command_handler(struct mbuf *mb, void *arg)
 
 	resp->pos = NETSTRING_HEADER_SIZE;
 
+	if (oe_dat)
+		sscanf(oe_dat->u.str, "%p", &data);
```

Comment:
is it possible to use `re_regex` instead ?

sscanf is not secure if I remember correctly.

Reply:
I think that we will not need the change in ctrl_tcp because we should not set the carg->data.